config.py

```diff
@@ -24,6 +24,7 @@ class BaseConfig:
         ('pl', 'Polszczyzna'),
     )
     CATEGORIES = ['impact', 'experience', 'raw', 'creative']
+    STATIC_PAGES_SLUGS = ['about', 'contact', 'vision', 'faq', 'terms']
```

Comment:
Note that slugs are dynamic from the title, so I don't know how to handle this from here except maybe referencing all potential slugs in all languages? 😓 Any idea @vinyll?